Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -272,7 +273,9 @@ namespace Multiplayer
         auto prefabEditorEntityOwnershipInterface = AZ::Interface<AzToolsFramework::PrefabEditorEntityOwnershipInterface>::Get();
         if (!prefabEditorEntityOwnershipInterface)
         {
-            AZ_Error("MultiplayerEditor", prefabEditorEntityOwnershipInterface != nullptr, "PrefabEditorEntityOwnershipInterface unavailable");
```

Comment:
Could you give some context how these changes are related to the rest of the PR please? I'm not immediately clear on that.

Reply:
It's the changes that Alex did, I don't have context, but it seems like he is using now the correct boolean to check if prefabs are enabled or not. I've included sig-networking so they can take a look.

Reply:
Sorry I'm quite late for this, but can give some insight.
This always errored when having prefabs disabled, which is not correct and made the test to fail.

If we are in prefab enabled mode and the interface does not exist, then that is an error.
But in slice mode, is expected that this interface should not exist and that should *not* be an error.

In the future if you need more info on anything, just message me on discord :)